cfme/tests/intelligence/reports/test_metering_report.py

```diff
@@ -1,4 +1,8 @@
 # -*- coding: utf-8 -*-
+# All providers that support C&U support Metering Reports.SCVMM doesn't support C&U.
```

Comment:
@nachandr I don't think you want this commit in this PR :)

Reply:
@mshriver , done